### DISCUSSION TITLE
STM32F105xxx & STM32F107xxx NOT define in board list

### DISCUSSION THREAD

**Discussion by pouria721** [stm32duino/Arduino_Core_STM32#2104] [General]
why STM32F105xxx & STM32F107xxx there are in variant folder but not defined in board list?

**Comment by fpistm** [comment]
Since STM32 core v2.0.0, all variants files are generated for all STM32 mcu series. Anyway, it required to add linkerscript and declare them in the boards.txt.
https://github.com/stm32duino/Arduino_Core_STM32/wiki/Add-a-new-variant-%28board%29
